src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,4 +121,16 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Parses a {@code String fileLocation} into a {@code FileLocation}.
```

Comment:
Instead of returning String, should be returning a FileLocation based on your javadoc comment; but I don't believe there is a need to define a new class for this, just return a java.nio.Path as this is what the rest of the program uses for File Parsing (refer to XmlSerializableAddressBookTest.java or Alex's v1.1 for example of file parsing). Please change return type to Path and the JavaDoc accordingly. 

(You can choose another abstraction instead of Path e.g. File, I recommend Path though, but we don't return a String, because then the parser is not actually doing anything; it takes a string and returns a string. In other instances used by ParserUtil, it always returns an Object from the String, not a String.